plugins/google/client.js

```diff
@@ -5,13 +5,15 @@ function onGoogleSignIn(googleUser) {
 	};
 
 	var xhttp = new XMLHttpRequest();
-	xhttp.onreadystatechange = () => {
+	xhttp.onreadystatechange = function() {
```

Comment:
greater browser compatibility and changes scope so this.readyState can be accessed.